docs(site): Add FreeBSD port/package installation instructions (#2237)
kopia/kopia: site/content/docs/Installation/_index.md

To build Kopia from ports yourself, cd /usr/ports/sysutils/kopia and follow the [Ports](https://www.openbsd.org/faq/ports/ports.html) guide on building ports as usual.

#### FreeBSD installation via ports

FreeBSD now has kopia in ports, which means it gets built as packages in snapshots for several platforms (amd64, arm64 and i386) and will appear as a package for supported versions.

To install the port:

```shell

> # cd /usr/ports/sysutils/kopia/ && make install clean

Reviewer: ```suggestion
cd /usr/ports/sysutils/kopia/ && make install clean
```

```

To add the package, run one of these commands:

```shell

> # pkg install sysutils/kopia
> # pkg install kopia

Reviewer: ```suggestion
pkg install sysutils/kopia
pkg install kopia
```

```

> More info on [ports](https://docs.freebsd.org/en/books/handbook/ports/index.html#ports) FreeBSD Handbook guide.

Reviewer: ```suggestion
For more information on ports, see the [FreeBSD Handbook](https://docs.freebsd.org/en/books/handbook/ports/index.html#ports).
```
